app/assets/javascripts/chessboard.js

```diff
@@ -0,0 +1,103 @@
+$(function() {
+    // draw the chess board
+    for (var y = 0; y <= 7; y++) {
+        $("#chess-board").append(
+          "<tr>" +
+            "<td data-x='0' data-y='" + y + "'></td>" +
+            "<td data-x='1' data-y='" + y + "'></td>" +
+            "<td data-x='2' data-y='" + y + "'></td>" +
+            "<td data-x='3' data-y='" + y + "'></td>" +
+            "<td data-x='4' data-y='" + y + "'></td>" +
+            "<td data-x='5' data-y='" + y + "'></td>" +
+            "<td data-x='6' data-y='" + y + "'></td>" +
+            "<td data-x='7' data-y='" + y + "'></td>" +
+          "</tr>");
+    }
+
+    // assign variables to initial positions based on
+    // their data attributes
+    // black piece variables
+    var blackRookLeft = 'td[data-x="0"][data-y="0"]';
+    var blackKnightLeft = 'td[data-x="1"][data-y="0"]';
+    var blackBishopLeft = 'td[data-x="2"][data-y="0"]';
+    var blackQueen = 'td[data-x="3"][data-y="0"]';
+
+    var blackKing = 'td[data-x="4"][data-y="0"]';
+    var blackBishopRight = 'td[data-x="5"][data-y="0"]';
+    var blackKnightRight = 'td[data-x="6"][data-y="0"]';
+    var blackRookRight = 'td[data-x="7"][data-y="0"]';
+
+    // white piece variables
+    var whiteRookLeft = 'td[data-x="0"][data-y="7"]';
+    var whiteKnightLeft = 'td[data-x="1"][data-y="7"]';
+    var whiteBishopLeft = 'td[data-x="2"][data-y="7"]';
+    var whiteQueen = 'td[data-x="3"][data-y="7"]';
+
+    var whiteKing = 'td[data-x="4"][data-y="7"]';
+    var whiteBishopRight = 'td[data-x="5"][data-y="7"]';
+    var whiteKnightRight = 'td[data-x="6"][data-y="7"]';
+    var whiteRookRight = 'td[data-x="7"][data-y="7"]';
```

Comment:
IMO you need a more dynamic data structure here. Lots of repetition, when something like (pseudo): `{white: {class: 'white-pcs', pieces: [{class: 'glyphicon-tower', id: 1, name: 'Rook'}], black: ...}` would work